.github/ISSUE_TEMPLATE/bug-report-advanced

```diff
@@ -0,0 +1,74 @@
+name: "\U0001F41E Bug report"
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before submitting a bug report**
+
+        The issue list is reserved for bug reports and feature requests. If you have a usage question, you can:
+
+        - Read the [documentation](https://blazorise.com/docs)
+        - Ask questions on [Discord](https://discord.io/blazorise)
+
+        Also, try to search for your issue/feature - another user may have already requested something similar; Thanks!
+
+  - type: input
+    id: version
+    attributes:
+      label: Blazorise Version
```

Comment:
The Blazorise Provider might be good to add. Preferably if it's a dropdown of some sorts with our current providers.

Reply:
It is better to have an input so that users can write the exact version, eg 1.2.6.

Reply:
Who said anything about changing the input?

> The **Blazorise Provider** might be good to add. Preferably if it's a **dropdown** of some sorts with our current **providers**.

I.E

Bootstrap
Bootstrap5
Material

etc...

Reply:
Ahhhh. Since you were linking "Blazorise Version" I understood you want to have an option for 1.1, 1.2, 1.3, etc. Yeah, we can have an additional field for providers.